internal/rbd/controllerserver.go

```diff
@@ -1340,10 +1340,10 @@ func (cs *ControllerServer) doSnapshotClone(
 		return nil, err
 	}
 
+	// update rbd image name
+	rbdSnap.RbdImageName = cloneRbd.RbdImageName
```

Comment:
Instead of logging `rbdSnap`, why not log `cloneRbd`?

Or, would it not be cleaner to set  `RbdImageName` inside `cloneRbd.createSnapshot()`?

Reply:
> Or, would it not be cleaner to set RbdImageName inside cloneRbd.createSnapshot()?

looks good to me, this PR got merged. Will raise another PR for the same.